src/websocket/broadcast/gameBroadcast.ts

```diff
@@ -164,7 +164,12 @@ export const broadcastInGameUpdate = async (
   return usersInGame;
 };
 
-export const broadcastGameStart = async (ws: WebSocketClient, gameId: string, usersInGame: User[]): Promise<void> => {
+export const broadcastGameStart = async (
+  ws: WebSocketClient,
+  gameId: string,
+  hostConnectionId: string,
+  usersInGame: User[],
+): Promise<void> => {
```

Comment:
hostConnectionId looks fixed in initGameState, but not used in broadGameStart

Reply:
kk, will fix it